.github/actions/compute-sha256/src/main.ts

```diff
@@ -0,0 +1,21 @@
+import * as core from "@actions/core";
+import * as fs from "fs";
+import * as crypto from "crypto";
+
+function shasum256(untrustedPath: string): string {
+  if (!fs.existsSync(untrustedPath)) {
+    throw new Error(`File ${untrustedPath} not present`);
+  }
+  const untrustedFile = fs.readFileSync(untrustedPath);
+  return crypto.createHash("sha256").update(untrustedFile).digest("hex");
+}
+
+async function run(): Promise<void> {
+  // Get the path to the untrusted file from ENV variable 'UNTRUSTED_PATH'
+  const untrustedPath = core.getInput("untrusted_path");
```

Comment:
the input name is called "path", not "untrusted_path". I think you need to update the string?

Reply:
I have updated it. thanks